tests/test_customization.py

```
@@ -11,54 +11,49 @@
 
 @pytest.fixture
 def _mock_graphql():
-    with patch("unraid_mcp.tools.customization.make_graphql_request", new_callable=AsyncMock) as m:
+    with patch("unraid_mcp.tools.unraid.make_graphql_request", new_callable=AsyncMock) as m:
         yield m
 
 
 def _make_tool():
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `_make_tool`

(ANN202)